backend/generating/Main.java

```diff
@@ -0,0 +1,28 @@
+package generation;
+
+import java.io.IOException;
+import java.util.LinkedList;
+
+public class Main {
+
+    public static void main(String[] args) throws IOException {
+
+        //temporarily set to null until client side stuff comes in
+        String studentName = null ;
+        String studentEmail = null;
+        String svrName = null;
+        String svVenue = null;
+        String svCourse = null;
+        String svNumber = null;
+        String svDate = null;
+        String svTime = null;
+
+        SupervisionInfo supervisionInfo = new SupervisionInfo(studentName, studentEmail, svrName, svVenue,svCourse, svNumber, svDate, svTime);
+        SupervisionQuestions questionList = new SupervisionQuestions(new LinkedList<Question>());

```

Comment:
Can't see SupervisionQuestions in this pull request.

Reply:
Actually, in a previous attempt, we called `SupervisionQuestions` a `SupervisionAgenda` and we called a `Question` a `Task`. I quite like that naming convention.